app/controllers/bookings_controller.rb

```diff
@@ -4,6 +4,7 @@ def index
     # @booking = Booking.find(params[:id])
 
     # @course = Course.find(params[:id])
+    Booking.booking_count
```

Comment:
place this method after create, and render message in terminal with total booking count.